components/sidebar/sidebar_channel/sidebar_channel.tsx

```diff
@@ -173,11 +180,12 @@ export default class SidebarChannel extends React.PureComponent<Props, State> {
                     index={channelIndex}
                 >
                     {(provided, snapshot) => {
+                        const {onTransitionEnd: draggableTransitionEnd, ...draggableProps} = provided.draggableProps;
                         return (
                             <li
                                 draggable='false'
                                 ref={this.setRef(provided.innerRef)}
-                                className={classNames('SidebarChannel', {
+                                className={classNames('SidebarChannel animating', {
```

Comment:
the word animating is misnomer in this context, if you see https://github.com/mattermost/mattermost-webapp/blob/9c8d80bfa9e03282325894b25c8438b1148c9760/sass/layout/_sidebar-left.scss#L1099-L1102
it just adds the transition property. better name should have been `animate` `transition` but i didnt wanted to change that in this pr